src/components/serviceRequest/ServiceInformationSection.tsx

```diff
@@ -111,15 +112,16 @@ export default function ServiceInformationSection(props: ServiceInformationSecti
             && <div className="grid col-12">
               <div className="col-fixed mr-3">{serviceInformationLabels.Source}</div>
               <div className="flex flex-wrap gap-3">
-                {Object.keys(sourceOptions).map((key, i) => (
+                {Object.keys(sourceOptions).map((label, i) => (
```

Comment:
just curious, why not "key"?

Reply:
I wanted the loop over sourceOptions to match the loops over priorityOptions and statusOptions. Those latter two are just arrays of labels rather than a key:value collection like sourceOptions. Their loop looks like statusOptions.map(label => .....) so I wanted this one to match. 

But in any case, these hardcoded options will be replaced by the app constants you've just introduced.

Reply:
Opps, actually looks like that statusOptions.map(label => ...) code wasn't included in this PR. I'll revert this change for now, since it will be repalced with the constants later anyway.